src/style.css

```diff
@@ -0,0 +1,155 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  padding: 0;
+  border: 0;
+}
+
+input {
+  outline: none;
+}
+
+.form-big {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin: 20px 0;
+  padding: 0;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form-big:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.search-big {
+  width: 100%;
+  padding: 0 0 0 62px;
+  height: 100%;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border: 0;
+}
+
+.search-big:hover {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-big:focus {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.form-small {
+  position: relative;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  height: 42px;
+  margin: 20px 0;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
you have a lot of similar styles with `form-big`
please check it and use DRY principle (Don't Repeat Yourself)

Reply:
and apply this principle to each similar classes like

```
.class1,
.class2 {
  background: red;
}

.class1 {
  width: 100px;
}

.class2 {
  width: 200px;
}
```